ideascube/serveradmin/backup.py

```diff
@@ -25,6 +24,7 @@ def make_name(format=None):
 class Backup(object):
 
     FORMAT = settings.BACKUP_FORMAT
+    SUPPORTED_FORMATS_AT_CREATION = ('bztar', 'gztar', 'tar')
```

Comment:
Since the code is there, it causes no difficulty to keep it, and there might be existing backups we don't know about, I'd say no, let's keep it, at least for some time, to provide a safer transition path.

How about we remove it in a month or two, when we can feel confident that people either have made new tar backups, or they just don't do backups at all?